File: 프로그래머스/0/181847. 0 떼기/0 떼기.js
function solution(n_str) {
  let num = 0;
  for (let i = 0; i < n_str.length; i++) {
    if(n_str[i] == 0) {
      num += 1
    } else break
  }
  return num != 0 ? n_str.slice(num) : n_str
}

File: 프로그래머스/0/181847. 0 떼기/0 떼기.java
class Solution {
    public String solution(String n_str) {
        int num = 0;
        
        for (int i = 0; i < n_str.length(); i++) {
            if (n_str.charAt(i) == '0') {
                num += 1;
            } else {
                break;
            }
        }
        
        return num !=0 ? n_str.substring(num) : n_str;
    }
}